controllers/datadogagent/agent_test.go

```diff
@@ -824,6 +872,29 @@ func defaultPodSpec() corev1.PodSpec {
 	}
 }
 
+func appendDefaultAPMAgentContainer(podSpec *corev1.PodSpec) {
+	apmContainer := corev1.Container{
+		Name:            "trace-agent",
+		Image:           "datadog/agent:latest",
+		ImagePullPolicy: corev1.PullIfNotPresent,
+		Command:         []string{"trace-agent", "--config=/etc/datadog-agent/datadog.yaml"},
+		Resources:       corev1.ResourceRequirements{},
+		Ports:           []corev1.ContainerPort{{Name: "traceport", ContainerPort: 8126, Protocol: "TCP"}},
+		Env:             defaultAPMContainerEnvVars(),
+		VolumeMounts: []corev1.VolumeMount{
+			{Name: "config", MountPath: "/etc/datadog-agent"},
+			{
+				Name:      "custom-datadog-yaml",
+				ReadOnly:  true,
+				MountPath: "/etc/datadog-agent/datadog.yaml",
```

Comment:
I feel like we should have these as consts, it's repeated in another form above.

Reply:
the const value exist in `api/v1alpha1`. but the idea here was to no use it in the test

Reply:
I can add another const value in the test file.